aiosmtpd/tests/test_smtp.py

```diff
@@ -105,7 +105,8 @@ def authenticate(
 
 class DecodingControllerPeekAuth(Controller):
     def factory(self):
-        return Server(self.handler, decode_data=True, enable_SMTPUTF8=True,
+        self.server_kwargs["enable_SMTPUTF8"] = True
+        return Server(self.handler, decode_data=True,
```

Comment:
`Controller` now injects `enable_SMTPUTF8` into `server_kwargs`, so if we want to ensure `enable_SMTPUTF8=True`, instead of passing it to `SMTP` via explicit keyword arg, we must override the value in `server_kwargs`.